docs/how_to/menus.rst

```diff
@@ -175,26 +178,31 @@ To make your modifier available, it then needs to be registered with
 Now, when a page is loaded and the menu generated, your modifier will
 be able to inspect and modify its nodes.
 
-A simple modifier looks something like this::
+Here is an example of a simple modifier that places a Page's attribute in the corresponding
+``NavigationNode``::
 
     from menus.base import Modifier
     from menus.menu_pool import menu_pool
 
+    from cms.models import Page
+
     class MyMode(Modifier):
         """
 
         """
         def modify(self, request, nodes, namespace, root_id, post_cut, breadcrumb):
+            # if the menu is not yet cut, don't do anything
             if post_cut:
                 return nodes
-            count = 0
+            # otherwise loop over the nodes
             for node in nodes:
-                node.counter = count
-                count += 1
+                # does this node represent a Page?
+                if node.attr["is_page"]:
+                    # if so, put its changed_by attribute on the node
+                    node.changed_by = Page.objects.get(id=node.id).changed_by
```

Comment:
@czpython This is nothing compared to the queries that I've put on menus! When you need an attribute, you need it.

Why should we encourage the use of ``attr``? Is it just to avoid potential conflicts with the names of existing attributes?